fix: update docs on translation contributions

emmambd opened the pull request:
We want to welcome translation contributions, while being clear that any contributions may be subject to being overwritten in the future when there are documentation changes.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/MobilityData/gtfs.org?pullRequest=85) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/MobilityData/gtfs.org?pullRequest=85) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/MobilityData/gtfs.org?pullRequest=85) it.</sub>

fabrice-v commented:
Thks, will merge shortly!

emmambd committed:
fix: update docs on translation contributions

We want to welcome translation contributions, while being clear that any contributions may be subject to being overwritten in the future when there are documentation changes.
